linode_api4/objects/networking.py

```diff
@@ -9,20 +9,40 @@ class IPv6Pool(Base):
     properties = {
         "range": Property(identifier=True),
         "region": Property(slug_relationship=Region, filterable=True),
+        "prefix": Property(),
+        "route_target": Property(),
     }
 
+    def _api_get(self):
+        """
+        A helper method to GET this object from the server
+        """
+        pools = self._client.networking.ipv6_pools()
+        pool = [p for p in pools if p.range == self.range][0]
```

Comment:
Checking whether `[p for p in pools if p.range == self.range]` is an empty list before invoke subscript (`[0]`) might be safer here.

Reply:
By the way, it seems IPv6 pool is [entirely deprecated](https://www.linode.com/docs/products/compute/compute-instances/guides/network-infrastructure-upgrades/#whats-changing)? And there is no longer an API to create one?

Reply:
Thanks for the tip! I'm not sure what to do about the fact that IPv6 pool is deprecated now. The ticket specified updating the fields in this object and I don't think we should remove it and its functionality entirely. What do you think we should do about this?

Reply:
Since it is deprecated now and we can't create a new pool, which means we can't test it any more... I would suggest leaving it unchanged and add a deprecation mark in the doc string.

Reply:
I'd say it's probably fine to add a deprecation note to the docstring here and avoid making any changes. FWIW we can unit test this functionality using mocking, but it might not be worth pursuing given the imminent removal.

Reply:
Agreed. I removed the changes to IPv6Pool and added a deprecation notice.